ndk/src/events/kinds/lists/index.ts

```diff
@@ -68,6 +101,7 @@ export class NDKList extends NDKEvent {
 
     /**
      * Sets the name of the list.
+     * @deprecated Please use "title" instead.
```

Comment:
Should I also go ahead and mimic the behaviour of the set title method here too so that we're pushing events to the right structure?